modules/service/src/main/scala/config/Config.scala

```diff
@@ -85,7 +85,7 @@ object Config {
       keyPair.getPublic,
       keyPair.getPrivate,
       8080,
-      "lucuma.xyz",
+      "local.lucuma.xyz",
```

Comment:
This is only for local development, which assumes `localhost` is aliased to `local.lucuma.xyz`.